client/src/app/routes/stardust/AddressState.ts

```diff
@@ -51,6 +54,9 @@ export interface IAddressState {
     tokensCount: number;
     nftCount: number;
     associatedOutputCount: number;
+    aliasContainsDID: boolean;
```

Comment:
Please add "defaults" for the new fields in `initialState`. I suggest making the type of resolvedDID `IIdentityStardustResolveResponse | null` and make it `null` by default for easier checks if it's populated.

Reply:
added the default values 💪